pages/advanced/cache.en-US.md

```diff
@@ -82,13 +82,13 @@ export default function App() {
 }
 ```
 
-### Example
+### Example: Mutate Multiple Keys
```

Comment:
how about use `h2` for `Examples` and use `h3` for `Mutate Multiple Keys` under it

Reply:
Sounds great!